test/HealthChecks.OpenIdConnectServer.Tests/HealthChecks.OpenIdConnectServer.approved.txt

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class IdSvrHealthCheckBuilderExtensions
     {
+        public static Microsoft.Extensions.DependencyInjection.IHealthChecksBuilder AddIdentityServer(this Microsoft.Extensions.DependencyInjection.IHealthChecksBuilder builder, System.Func<System.IServiceProvider, System.Uri> uriProvider, string? name = null, Microsoft.Extensions.Diagnostics.HealthChecks.HealthStatus? failureStatus = default, System.Collections.Generic.IEnumerable<string>? tags = null, System.TimeSpan? timeout = default) { }
         public static Microsoft.Extensions.DependencyInjection.IHealthChecksBuilder AddIdentityServer(this Microsoft.Extensions.DependencyInjection.IHealthChecksBuilder builder, System.Uri idSvrUri, string? name = null, Microsoft.Extensions.Diagnostics.HealthChecks.HealthStatus? failureStatus = default, System.Collections.Generic.IEnumerable<string>? tags = null, System.TimeSpan? timeout = default) { }
     }
-}
\ No newline at end of file
+}
```

Comment:
```suggestion
}

```

Reply:
My editor kept insisting on adding the empty line. `cp` to the rescue.